website/content/en/docs/component-guides/kserve/access-aws-service-from-kserve.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "Access AWS Service from KServe"
+description = "Configuration for accessing AWS services for inference services such as pulling images from private ECR and downloading models from S3 bucket."
```

Comment:
action words language is different for other docs, this should start with `Configure inference services to `